src/components/PathwayExercise/index.js

```diff
@@ -32,11 +32,13 @@ import ExerciseImage from "./ExerciseImage/ExerciseImage.js";
 import { breakpoints } from "../../theme/constant";
 
 const languageMap = {
-  hi: "Hindi",
+  "hi-IN": "Hindi",
   en: "English",
-  te: "Telugu",
+  "te-IN": "Telugu",
   ta: "Tamil",
   mr: "Marathi",
+  "or-IN": "Oriya",
+  "kn-IN": "Kannada",
 };
```

Comment:
Yeh `languageMap` alag jagah upayog karega kya? Agar hai, toh hum `constants` file mein yeh daal sakte hain. Lekin yahan (https://github.com/navgurukul/bhanwari-devi/blob/main/src/constant.js#L160-L166) hamare paas ek `lang` object User Interface language ke liye. Hum is object se codes badal sakte hain, lekin phir, hamen aur badlav banana zarooree hai (e.g., renaming the file and constants in https://github.com/navgurukul/bhanwari-devi/blob/main/src/msg/hi.js).